ambry-store/src/test/java/com.github.ambry.store/LogSegmentTest.java

```diff
@@ -69,6 +71,7 @@ public void cleanup() {
       }
     }
     assertTrue("The directory [" + tempDir.getAbsolutePath() + "] could not be deleted", tempDir.delete());
+    validateMockitoUsage();
```

Comment:
Why is this here?

Reply:
Quote from: https://static.javadoc.io/org.mockito/mockito-core/2.9.0/org/mockito/Mockito.html#validateMockitoUsage()
```
Sometimes though, you might want to validate the framework usage explicitly. For example, one of the users wanted to put validateMockitoUsage() in his @After method so that he knows immediately when he misused Mockito. Without it, he would have known about it not sooner than next time he used the framework. One more benefit of having validateMockitoUsage() in @After is that jUnit runner and rule will always fail in the test method with defect whereas ordinary 'next-time' validation might fail the next test method.
```